public/app/plugins/panel/nodeGraph/EdgeArrowMarker.tsx

```diff
@@ -12,7 +12,7 @@ export function EdgeArrowMarker() {
         viewBox="0 0 10 10"
         refX="8"
         refY="5"
-        markerUnits="strokeWidth"
+        markerUnits="userSpaceOnUse"
```

Comment:
Stops the arrow head from increasing on hover. Felt like the effect was too strong.